.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
I think monthly would be enough, otherwise, we're going to overwhelmed by the notifications and PRs.

Reply:
@anapaulagomes great suggestion, Ana :) We are not in a bleeding edge rush here. 😊